pcp/Platform.c

```diff
@@ -699,6 +703,16 @@ bool Platform_getNetworkIO(NetworkIOData* data) {
    return true;
 }
 
+void Platform_getFileDescriptors(double* used, double* max) {
+   *used = NAN;
```

Comment:
Need to initialize max here... may as well use what @BenBE used for Linux.

`*max = 65536;`